nupic/algorithms/KNNClassifier.capnp

```diff
@@ -0,0 +1,41 @@
+@0x99e493bfb8fbad94;
+
+# Next ID: ???
```

Comment:
It should be the next ID available in the following struct. In this case, it would be 31.